index.js

```diff
@@ -22,7 +28,16 @@ function getPercentageScore(score) {}
  * @param {object} score2 - An object representing another score of an assignment. Has the same shape as `score1`.
  * @returns {number} A rounded integer, representing the highest percentage scored among the two given scores.
  */
-function getHighestOfTwoScores(score1, score2) {}
+function getHighestOfTwoScores(score1, score2) {
```

Comment:
While this is a phenomenal step-by-step approach, you've already got the logic in the previous function. How could you use that function in this one?